packages/roosterjs-content-model-dom/lib/modelToDom/handlers/handleBlockGroupChildren.ts

```diff
@@ -39,6 +40,14 @@ export const handleBlockGroupChildren: ContentModelHandler<ContentModelBlockGrou
             }
         });
 
+        if (
+            group.blockGroupType == 'Document' &&
+            group.format &&
+            isNodeOfType(parent, 'ELEMENT_NODE')
+        ) {
+            applyFormat(parent, context.formatAppliers.segment, group.format, context);
```

Comment:
format on Document is not used for applying to the root, but it is used for default segment format

Reply:
What is the scenario to change the default format after content model is created?

Reply:
In that case, you need to reformat the whole doc where user typed. Because this format is applied everytime user is typing, but not used for formatting the doc.

Reply:
Hmm, good point. Let me confirm with Jorge.
I guess another option is to let the content just before the user changed the mailbox, but when they try to Clear format, use the new account default format.

Reply:
anyway, I think the fix should be in OWA side

Reply:
Hmm, if that is the case, sure I can do the change in OWA, but since there are no capabilities from Content Model to do it, might need to do it modifying the DOM directly, which defeats the purpose of having content model.

Also, from formatContentModel API perspective, modifying the format of the model is not doing anything. So I thought that was a bug. Maybe we can set that property as readonly so users are not able to modify it.